celest/encounter/__init__.py

```diff
@@ -1,3 +1 @@
 
```

Comment:
Add an import statement for `generate_vtw`.

```suggestion
from celest.encounter.window_generator import generate_vtw
```